LICENSE.md

```diff
@@ -1,25 +1,210 @@
-Copyright (c) IBM Corp. 2016. All Rights Reserved.
+Copyright (c) IBM Corp. 2014,2016. All Rights Reserved.
 Node module: strong-error-handler
-This project is licensed under the MIT License, full text below.
+This project is licensed under the Artistic License 2.0, full text below.
```

Comment:
What is the rationale for changing the license from MIT to AL-2.0? I believe most loopback modules use MIT.